lxc/list.go

```diff
@@ -53,12 +55,24 @@ A single keyword like "web" which will list any instance with a name starting by
 A regular expression on the instance name. (e.g. .*web.*01$).
 A key/value pair referring to a configuration item. For those, the
 namespace can be abbreviated to the smallest unambiguous identifier.
+A key/value pair where the key is a shorthand. Multiple values must be delimited by ','. Available shorthands:
+  - type={'CONTAINER','VIRTUAL-MACHINE'}
+  - status={'RUNNING','STOPPED','FROZEN'}
+  - architecture={cpu-arch|string}
+  - device={device name|string}
```

Comment:
I'm not convinced about the need for the `device` filter, it's probably best to skip it at this point and reconsider it when there's a clearer need for something like it.